backend/src/middleware/hashtags/hashtagsMiddleware.spec.js

```diff
@@ -36,7 +36,7 @@ beforeAll(() => {
     context: () => {
       return {
         user: authenticatedUser,
-        neode: instance,
+        neode: neode,
```

Comment:
```suggestion
        neode,
```
can't we use ES6 syntax here?